.github/actions/install-metal-dev-deps/action.yml

```diff
@@ -43,3 +43,4 @@ runs:
         wget https://apt.llvm.org/llvm.sh
         chmod u+x llvm.sh
         sudo ./llvm.sh 17
+        sudo apt install libc++-17-dev libc++abi-17-dev
```

Comment:
Does `llvm.sh` add repositories to the system `apt`?

Reply:
Yep